marklogic-data-hub/src/server-side/impl/trace-lib.xqy

```diff
@@ -302,7 +302,7 @@ declare %private function trace:write-error-trace(
             "/" || $trace/*:trace/*:traceId || $extension,
             $trace,
             xdmp:default-permissions(),
-            ("trace", $trace/*:trace/*:type)
+            ("trace", $trace/*:trace/*:flow-type, $trace/*:trace/*:flowType)
```

Comment:
Two test failures -- assertions around binary nodes.  this bugfix looks legit, the tests' output has changed in what looks like a non-problematic way.  but they need to change to do the merge